source/screens/form/FormCaseScreen.tsx

```diff
@@ -26,8 +27,12 @@ import type {
   Answer,
 } from "../../types/Case";
 
+const FROM_LOADING_TO_FORM_DELAY = 500;
+
 const FormCaseScreen = ({ route, navigation }: Props): JSX.Element => {
   const [encryptionPin, setEncryptionPin] = useState("");
+  const [loadingEncryptionPin, setLoadinEncryptionpin] = useState(true);
```

Comment:
No G in "loading", or capital P in "pin" for `setLoadinEncryptionpin` - literally unusable code I'm afraid.